### DISCUSSION TITLE
Two PG_USER variable in the pgsync.com website

### DISCUSSION THREAD

**Discussion by vijeandran** [toluaina/pgsync#276] [Show and tell]
Hi toluaina,

On the pgsync website (https://pgsync.com/env-vars/), there are two PG_USER variables. I think there is a typo error.
Please do changes. Also, add variables for Redis host and port.

export PG_USER=kermit-the-frog
export PG_USER=localhost
export PG_PORT=5432

Thanks,

**Comment by toluaina** [comment]
Thanks this has been fixed. The redis variables are listed on that page as well when you scroll down
